alf/networks/rnn.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2020 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import gin
+import torch
+import torch.nn as nn
+
+import alf
+from .network import Network
+
+
+@gin.configurable(whitelist=['output_layers'])
+class StackedLSTMCell(Network):
```

Comment:
Can this be integrated into LSTMEncodingNetwork?

Reply:
integrated.